src/Expensify.js

```diff
@@ -62,7 +62,8 @@ const propTypes = {
     /** Whether a new update is available and ready to install. */
     updateAvailable: PropTypes.bool,
 
-    /** Tells us if the sidebar has rendered */
+    /** Tells us if the sidebar has rendered - TODO: We don't use it as temporary solution to fix not hidding splashscreen */
```

Comment:
could you clarify the TODO comment? Usually we don't leave TODO comments. If we do, it should be clear to everyone what is left to do.